dataactvalidator/scripts/loadTas.py

```diff
@@ -126,11 +214,38 @@ def existing_id(row, existing):
         Args:
             row: row to check in
             existing: Dict[account_num, List[TASLookup]]
+        Returns:
+            account number for existing row
     """
     for potential_match in existing[row['account_num']]:
         return potential_match.account_num
 
 
+def missing_records(sess):
+    """ Pull in the empty records
+
+        Args:
+            sess: connection to the database
+        Returns:
+            list of account numbers representing records with empty values
+    """
+    empty_records = sess.query(TASLookup).filter_by(budget_function_code=None).values('account_num')
+    return [int(empty_record[0]) for empty_record in empty_records]
+
+
+def get_parser():
+    """ Generates list of command-line arguments
+
+        Returns:
+            argument parser to be used for commandline
+    """
+    tas_parser = argparse.ArgumentParser(description='Import data from the cars_tas.csv')
+    tas_parser.add_argument('--backfill_historic', '-b', action='store_true', help='Backfill tas with historical data')
+    return tas_parser
+
 if __name__ == '__main__':
+    parser = get_parser()
```

Comment:
For consistency, in Broker we generally define the arg parser in the main function (whether it's called `main()` or something else), rather than creating a `get_parser` function. This is usually done outside this `if __name__` section and after logging is configured.

Not something that absolutely has to be changed, just something to remember for next time.

Reply:
Sounds good.